sp041_20191205T1757-instrument-scalar_v2.cdl

```diff
@@ -0,0 +1,417 @@
+		PHASE:phase_calculation_method_doi = "TBD";
+	byte PHASE_QC(N_MEASUREMENTS) ;
+		PHASE_QC:_FillValue = 0b ;
+		PHASE_QC:long_name = "quality flag" ;
+	double TIME(N_MEASUREMENTS) ;
+		TIME:_FillValue = -1. ;
+		TIME:long_name = "time of measurement and GPS location" ;
+		TIME:standard_name = "time" ;
+		TIME:calendar = "gregorian" ;
+		TIME:units = "seconds since 1970-01-01T00:00:00Z" ;
+		TIME:valid_min = 1e9 ;
+      TIME:valid_max = 4e9 ;
+		TIME:interpolation_methodology = "TBD" ;
+		TIME:interpolation_methodology_vocabulary = "TBD" ;
+		TIME:interpolation_methodology_doi = "TBD" ;
+	double LATITUDE(N_MEASUREMENTS) ;
+		LATITUDE:_FillValue = -9999.9 ;
+		LATITUDE:long_name = "latitude of each measurement and GPS location" ;
+		LATITUDE:standard_name = "latitude" ;
+		LATITUDE:units = "degrees_north" ;
+		LATITUDE:valid_max = "90" ;
+		LATITUDE:valid_min = "-90" ;
+		LATITUDE:interpolation_methodology = "TBD" ;
+		LATITUDE:interpolation_methodology_vocabulary = "TBD" ;
+      LATITUDE:interpolation_methodology_doi = "TBD" ;
+	double LONGITUDE(N_MEASUREMENTS) ;
+		LONGITUDE:_FillValue = -9999.9 ;
+		LONGITUDE:long_name = "longitude of each measurements and gps locations" ;
+		LONGITUDE:standard_name = "longitude" ;
+		LONGITUDE:units = "degrees_east" ;
+		LONGITUDE:valid_max = "180" ;
+		LONGITUDE:valid_min = "-180" ;
+      LONGITUDE:interpolation_methodology = "TBD" ;
+      LONGITUDE:interpolation_methodology_vocabulary = "TBD" ;
+      LONGITUDE:interpolation_methodology_doi = "TBD" ;
+	float PRES(N_MEASUREMENTS) ;
+		PRES:_FillValue = -9999.9 ;
+		PRES:long_name = "Pressure" ;
+		PRES:vocabulary = "http://vocab.nerc.ac.uk/collection/OG1/current/PRES/";
+		PRES:standard_name = "sea_water_pressure" ;
+		PRES:units = "dbar" ;
+		PRES:ancillary_variables = "PRES_QC" ;
+		PRES:comment = "Sea water pressure, equals 0 at sea-level" ;
+        PRES:sensor = "/CTD";
+	byte PRES_QC(N_MEASUREMENTS) ;
+		PRES_QC:_FillValue = 0b ;
+		PRES_QC:long_name = "quality flag" ;
+	float DEPTH(N_MEASUREMENTS) ;
+		DEPTH:_FillValue = NaN ;
+		DEPTH:long_name = "Depth" ;
+		DEPTH:vocabulary = "http://vocab.nerc.ac.uk/collection/OG1/current/DEPTH/";
+		DEPTH:standard_name = "depth" ;
+		DEPTH:units = "m" ;
+		DEPTH:ancillary_variables = "DEPTH_QC" ;
+		DEPTH:positive = "down" ;
+        DEPTH:sensor = "/CTD";
+	byte DEPTH_QC(N_MEASUREMENTS) ;
+		DEPTH_QC:_FillValue = 0b ;
+		DEPTH_QC:long_name = "quality flag" ;
+	float TEMP(N_MEASUREMENTS) ;
+		TEMP:_FillValue = -9999.9 ;
+		TEMP:long_name = "Sea Water Temperature" ;
+        TEMP:vocabulary = "http://vocab.nerc.ac.uk/collection/OG1/current/TEMP/";
+		TEMP:standard_name = "sea_water_temperature" ;
+		TEMP:units = "Celsius" ;
+		TEMP:ancillary_variables = "TEMP_QC" ;
+		TEMP:valid_max = 40. ;
+		TEMP:valid_min = -5. ;
+		TEMP:coverage_content_type = "physicalMeasurement" ;
+		TEMP:coordinates = "time lon lat depth" ;
+		TEMP:comment = "Sea temperature in-situ ITS-90 scale" ;
+        TEMP:sensor = "/CTD";
```

Comment:
This currently points to the old CTD group. Should point to the INSTRUMENT_CTD variable, i.e.: DEPTH:sensor = "INSTRUMENT_CTD".